src/tax-lot/tax-lot.repository.schema.ts

```diff
@@ -1,7 +1,17 @@
 import { z } from "zod";
-import { boroughEntitySchema, landUseEntitySchema } from "src/schema";
+import {
+  boroughEntitySchema,
+  landUseEntitySchema,
+  zoningDistrictEntitySchema,
+} from "src/schema";
 import { taxLotEntitySchema } from "src/schema/tax-lot";
 
+export const checkTaxLotByBblRepoSchema = z.object({
+  bbl: z.string().regex(RegExp("^([0-9]{10})$")),
+});
```

Comment:
The shape of the schema is correct. But, we can be DRYer. The checkTaxLotByBbl method is selecting the bbl field from the tax lot table, represented by the taxLotEntitySchema. We can model this with zod's [Pick](https://zod.dev/?id=pickomit) method. This also means that if we ever need to update the bbl regex, we only need to update the taxLotEntitySchema and the "check" schema will automatically get the new regex.

```suggestion
export const checkTaxLotByBblRepoSchema = taxLotEntitySchema.pick({ bbl: true });
```

Reply:
Did not know about pick, updated with your suggested changes 👍